assets/app/views/create/nextSteps.html

```diff
@@ -46,9 +46,9 @@ <h1 ng-if="!pendingTasks(tasks()).length && erroredTasks(tasks()).length">Comple
         <p>You may want to use the <code>oc</code> command line tool to help with troubleshooting. After <a target="_blank" href="{{'get_started_cli' | helpLink}}">downloading and installing</a> it, you can log in, switch to this particular project, and try some commands :</p>
 
         <pre class="code prettyprint">oc login {{loginBaseUrl}}
-  oc project {{projectName}}
-  oc build-logs -h
-  oc logs -h</pre>
+oc project {{projectName}}
+oc build-logs -h
```

Comment:
thanks for the heads up, i'll update this

On Mon, Feb 1, 2016 at 4:45 PM, Michail Kargakis <notifications@github.com>
wrote:

> In assets/app/views/create/nextSteps.html
> <https://github.com/openshift/origin/pull/6946#discussion_r51485771>:
>
> > @@ -46,9 +46,9 @@ <h1 ng-if="!pendingTasks(tasks()).length && erroredTasks(tasks()).length">Comple
> >          <p>You may want to use the <code>oc</code> command line tool to help with troubleshooting. After <a target="_blank" href="{{'get_started_cli' | helpLink}}">downloading and installing</a> it, you can log in, switch to this particular project, and try some commands :</p>
> >
> >          <pre class="code prettyprint">oc login {{loginBaseUrl}}
> > -  oc project {{projectName}}
> > -  oc build-logs -h
> > -  oc logs -h</pre>
> > +oc project {{projectName}}
> > +oc build-logs -h
>
> We have deprecated build-logs so I think it makes sense to not include it
> anywhere anymore
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/6946/files#r51485771>.
>